Update index.md (#67152)
grafana/grafana: docs/sources/setup-grafana/configure-security/configure-authentication/grafana-com/index.md

```

### Configure automatic login
To automatically log in to Grafana using GrafanaCom authentication, set the `auto_login` option to `true`. This skips the login screen.

> This setting is ignored if multiple auth providers are configured to use auto login.

Reviewer: ```suggestion
This setting is ignored if multiple auth providers are configured to use auto-login.
```

```
auto_login = true
```

## Skip organization role sync

> To prevent the synchronization of organization roles from Grafana.com, set the `skip_org_role_sync` option to `true`. This is useful if you want to manage the organization roles for your users from within Grafana.

Reviewer: ```suggestion
To prevent synchronizing organization roles from Grafana.com, set the `skip_org_role_sync` option to `true`. This is useful if you want to use Grafana to manage organization roles
```
